packages/passport-interface/src/SemaphoreSignatureIntegration.ts

```diff
@@ -89,6 +93,50 @@ export function openSignedZuzaluUUIDPopup(
   openPassportPopup(popupUrl, proofUrl);
 }
 
+/**
+ * Opens a passport popup to generate a Semaphore signature proof on the user's
+ * Zuzalu DB uuid and website referer, which can then be used to fetch user details
+ * from the passport server, and ensure that the sign in signature was meant for this
+ * website. Built specifically for Zuzalu apps.
+ *
+ * @param urlToPassportWebsite URL of passport website
+ * @param popupUrl Route where the usePassportPopupSetup hook is being served from
+ * @param originalSiteName Name of site requesting proof
+ */
+export function openSignedZuzaluSignInPopup(
+  urlToPassportWebsite: string,
+  popupUrl: string,
+  originalSiteName: string
+) {
+  const proofUrl = constructPassportPcdGetRequestUrl<
+    typeof SemaphoreSignaturePCDPackage
+  >(
+    urlToPassportWebsite,
+    popupUrl,
+    SemaphoreSignaturePCDPackage.name,
+    {
+      identity: {
+        argumentType: ArgumentTypeName.PCD,
+        pcdType: SemaphoreIdentityPCDPackage.name,
+        value: undefined,
+        userProvided: true,
+      },
+      signedMessage: {
+        argumentType: ArgumentTypeName.String,
+        userProvided: true,
+        value: undefined,
+      },
+    },
```

Comment:
do we need this middle part? isn't it already specified by `signIn: true` below?

Reply:
Yes, the way I've designed the system unfortunately means that each argument is needed. I can remove the `userProvided` and `value` fields to make it a little less verbose though.